pkg/termcorder/termcorder_test.go

```diff
@@ -0,0 +1,116 @@
+package termcorder_test
+
+import (
+	"bytes"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/haguro/termcord/pkg/termcorder"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewTermcording(t *testing.T) {
+	t.Parallel()
+	cfg := &termcorder.Config{}
+	tc, err := termcorder.NewTermcording(cfg)
+	assert.IsType(t, &termcorder.Termcording{}, tc)
+	assert.NoError(t, err)
+}
+
+func TestTermcordingFromFlags(t *testing.T) {
+	t.Parallel()
+	t.Run("Test setting flags", func(t *testing.T) {
+		os.Args = []string{"./termcord", "-h", "-q", "-k", "foo.txt", "bar", "baz"}
+		want := &termcorder.Config{
+			Filename:      "foo.txt",
+			CmdName:       "bar",
+			CmdArgs:       []string{"baz"},
+			Interactive:   false,
+			PrintHelp:     true,
+			QuietMode:     true,
+			LogKeystrokes: true,
+		}
+
+		got, err := termcorder.TermcordingFromFlags()
+
+		assert.NoError(t, err)
+		assert.Equal(t, want, got.Config)
+	})
+
+	t.Run("Test setting filename", func(t *testing.T) {
+		os.Args = []string{"./termcord", "foo.txt"}
+		want := "foo.txt"
+		shell, _ := os.LookupEnv("SHELL")
```

Comment:
If we don't care whether the key was in the environment or not, we would call `os.Getenv`, which just returns the value (no `ok`).